docs/design-ux/launch.md

```diff
@@ -0,0 +1,8 @@
+# Launch
+When users launch the app, it checks if there is a lockbook account on the device. If so, the app proceeds to naviation. Otherwise, it prompts for sign-up or import.
+
+If the user opts to create an account, they enter a name and click to submit. If the operation succeeds, the app proceeds to navigation. Otherwise, an unobstructive message appears to represent the failure and the user can retry.
```

Comment:
I think prior to creating an account the user should be made aware of a few things. I think the right way to think about this is that this will be a *ULTRA* readable TOS. It should be clear that they're signing up to manage their own key, they should probably import into another device asap (for redundancy) or save the account string somewhere. etc.

Reply:
todo: succinctly explain "TOS" aka self-managed keys are not recoverable